arcgis-ios-sdk-samples/Scenes/Display a scene/README.md

```diff
@@ -27,4 +27,4 @@ When opened, the sample will display a scene. Pan and zoom to explore the scene.
 
 ## Tags
 
-3D, basemap, elevation, scene, surface
+3D, basemap style, elevation, scene, surface
```

Comment:
Correct me if I'm wrong, but this should not be updated since it is a scene sample, right?
```suggestion
3D, basemap, elevation, scene, surface
```

Reply:
Thanks for catching that! Yes, that's right scene samples shouldn't be affected.